app/src/app/job_result/hybrid.rs

```diff
@@ -335,50 +336,54 @@ mod tests {
         assert!(!HybridJobResultAppImpl::_should_store(&job_result_data));
     }
 
-    async fn setup() -> Result<HybridJobResultAppImpl> {
+    fn setup() -> Result<HybridJobResultAppImpl> {
         // dotenv::dotenv().ok();
-        let rdb_module = setup_test_rdb_module().await;
-        let redis_module = setup_test_redis_module().await;
-        let repositories = Arc::new(HybridRepositoryModule {
-            redis_module,
-            rdb_module,
-        });
-        let id_generator = Arc::new(IdGeneratorWrapper::new());
-        let mc_config = infra_utils::infra::memory::MemoryCacheConfig {
-            num_counters: 10000,
-            max_cost: 10000,
-            use_metrics: false,
-        };
-        let worker_memory_cache =
-            infra_utils::infra::memory::new_memory_cache::<Arc<String>, Vec<Worker>>(&mc_config);
-        // let job_memory_cache =
-        //     common::infra::memory::new_memory_cache::<Arc<String>, Vec<Job>>(&mc_config);
+        let rdb_module = setup_test_rdb_module();
+        TEST_RUNTIME.block_on(async {
```

Comment:
use single static runtime through all tests (break sqlx pool in destoying runtime)